frontend/pages/admin/manage/users/create.vue

```diff
@@ -24,7 +23,7 @@
             :rules="[validators.required]"
           />
           <v-select
-            v-if="households"
+            :disabled="!selectedGroupId"
             v-model="newUserData.household"
```

Comment:
```suggestion
            v-model="newUserData.household"
            :disabled="!selectedGroupId"
```
Lint warning around here that `v-model` should come before `:disabled`